main_induction.py

```diff
@@ -26,56 +33,31 @@ class ModelPerformance:
 
 
 models = [
+    (GaussianNB, {}),
     (DecisionTreeClassifier, {}),
-    (DecisionTreeClassifier, {'max_depth': 20}),
     (RandomForestClassifier, {'n_estimators': 30}),
-    (RandomForestClassifier, {'n_estimators': 30, 'max_depth': 20}),
+    (RandomForestClassifier, {'n_estimators': 30, 'max_depth': 40}),
+    (RandomForestClassifier, {'n_estimators': 50}),
+    (RandomForestClassifier, {'n_estimators': 50, 'max_depth': 40}),
+    (RandomForestClassifier, {'n_estimators': 70}),
+    (RandomForestClassifier, {'n_estimators': 70, 'max_depth': 40}),
+    (NeuralNetworkClassifier, {'epochs': 50, 'batch_size': 100}),
 ]
```

Comment:
might be just me but i think it would be nice to define models declaratively like this for the submissions as well, with modifiers (feature subsets and rejection skew) included. Doesn't have to use this exact API params are just separated out like this here for the modifier combinations stuff

Reply:
Good idea, I'll try to implement something like this.